sql/nm_spawn_points.sql

```diff
@@ -1989,6 +1989,7 @@ INSERT INTO `nm_spawn_points` VALUES (17596507,47,-132.983,0.000,178.993);
 INSERT INTO `nm_spawn_points` VALUES (17596507,48,-140.219,0.000,170.706);
 INSERT INTO `nm_spawn_points` VALUES (17596507,49,-92.879,0.000,218.895);
 INSERT INTO `nm_spawn_points` VALUES (17596720,0,-155.000,19.000,244.000);
+INSERT INTO `nm_spawn_points` VALUES (17596628,1,158,0,20);
```

Comment:
Does he only have 1 spawn position on retail? if it does don't need the entry in this table, and can remove teh call for this form its script. Otherwise he should have several entries here. If this is only because you don't know what other positions he can spawn at, ignore this post - someone will update him with more pos's soon (I can send one of my minions on it)